components/function-controller/pkg/webhook/validation_mutation_test.go

```diff
@@ -40,49 +40,14 @@ func TestMutation(t *testing.T) {
 	}
 
 	// mutate function
-	functionCreateHandler.mutatingFunction(functionWithoutVisibilityLabel, rnInfo)
+	functionCreateHandler.mutatingFunction(function, rnInfo)
 
 	// ensure defaults are set
-	g.Expect(functionWithoutVisibilityLabel.ObjectMeta).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
+	g.Expect(function.ObjectMeta).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
 		"Name":      gomega.BeEquivalentTo("example-123"),
 		"Namespace": gomega.BeEquivalentTo("ns"),
-		"Labels": gomega.BeEquivalentTo(map[string]string{
-			"serving.knative.dev/visibility": string(serverlessv1alpha1.FunctionVisibilityClusterLocal),
-		}),
 	}))
-	g.Expect(functionWithoutVisibilityLabel.Spec).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
```

Comment:
you've deleted quite a lot of tests - could you tell why? if they no longer suit you needs then rewrite that part

Reply:
Please read variables `functionWithoutVisibilityLabel` and `functionWithVisibilityLabel` :P Those only were nedeed when we had logic for labels in webhooks, not in reconcile.